lib/ansible/playbook/task.py

```diff
@@ -276,6 +276,12 @@ def _validate_failed_when(self, attr, name, value):
         if not isinstance(value, list):
             setattr(self, name, [value])
 
+    def _validate_loop_control(self, attr, name, value):
```

Comment:
Could we set a default on the FA instead of adding this method? I think it would be as simple as `default=LoopControl`

Reply:
I had that initially and then I changed it but I don't remember why 🧐 I'll try that.

Reply:
class/objects should  validate their FAs when validated, we should generalize this and not make loop_control custom, that opens up having other keywords expand as 'class/object' entries w/o having to repeat custom validation for each

Reply:
Done. This PR is now a much simpler change.

@bcoca are you saying we should generalize this in this PR? Just setting `default=LoopControl` seems like it checks the `w/o having to repeat custom validation for each`.

Reply:
had not realized that using the class itself made the validation recursive, if so +1